src/components/Comments/Comment.js

```diff
@@ -22,7 +22,7 @@ import {getAllowAnonDmStatus} from '../../service/chat';
 import {removeWhiteSpace} from '../../utils/Utils';
 import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
 import {fonts, normalize, normalizeFontSize} from '../../utils/fonts';
-import {getCaptionWithLinkStyle} from '../../utils/string/StringUtils';
+import {getCaptionWithLinkStyle, getOfficialAnonUsername} from '../../utils/string/StringUtils';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of the function `getOfficialAnonUsername` is added. Ensure that this function is correctly implemented and tested.
```diff
+ import {getCaptionWithLinkStyle, getOfficialAnonUsername} from '../../utils/string/StringUtils';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->